health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -3757,6 +4018,67 @@
+        rowNumber,
+        sheetName: userSheet,
+        errorDetails: "Invalid username format User name can only be alphanumeric",
+      });
+    }
+
+    // Collect row numbers for duplicates
+    if (username) {
+      if (!usernameMap.has(username)) {
+        usernameMap.set(username, []);
+      }
+      usernameMap.get(username)!.push(rowNumber);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)